flatgeobuf/flatgeobuf.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>make the <code>fiona</code> paragraph a note/read more box.</p><p>I would rephrase our opinion - use <code>pyogrio</code> at the this time, and if you have new enough <code>gdal also use_arrow=True</code> (is this only for geoparquet?)  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
99% of people who install pyogrio will do it via the wheels, not from source, in which case the bundled GDAL will be of version 3.6+. I should probably reword that version paragraph.

> is this only for geoparquet?

No, it's a bit confusing from the naming, but it works for every format. (This is related to [this comment](https://github.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/25#discussion_r1297857919)). Originally pyogrio built the columnar arrays on its own in Cython, but now GDAL is able to build the columnar arrays _itself_ and pass those on to any environment that's able to understand Arrow arrays. So essentially the Arrow support _supplants_ the original pyogrio implementation, and will probably become the main method in the future because it's 50x less code (because it's all in the GDAL method)